docs/src/components/CardForm/CardForm.js

```diff
@@ -11,6 +11,7 @@ export default () => {
         onSubmit={e => console.log('onSubmit')}
         onCancel={e => console.log('onCancel')}
         onChange={e => console.log('onChange')}
+        acceptedCards={['JCB', 'VISA', 'MC']}
```

Comment:
Should we also add here UNIONPAY?

Reply:
The problem adding Unionpay is that it removes the validations for the card number input. This is the logic that we've right now implemented, so maybe is not the best thing for this example since we want the show the errors to the user too.